os.md

```diff
@@ -2,7 +2,8 @@
 
  Name | Version | [LSE Support](optimizing.md#locksynchronization-intensive-workload) | Kernel page size | AMI | Metal support | Comment
 ------ | ------ | ----- | ----- | ----- | ----- | -----
-Amazon Linux 2 | 2.26-35 or later| Yes | 4KB | [AMIs](amis_cf_sm.md) | Yes |
+Amazon Linux 2023 | All versions | Yes | 4KB | [AMIs](amis_cf_sm.md) | Yes | Improved performance over Amazon Linux 2
```

Comment:
> Improved performance over Amazon Linux 2

Given what I've seen from my team's work, I am skeptical. What data was used to make that claim?

Reply:
If I were going to call out a reason to upgrade, it would be related to how it's not ancient. But that's just me and certainly isn't approved official language :)

Reply:
I think we should use an internal source, like an AWS blog post instead of third party data.

Reply:
I've updated it to mention Pointer Authentication support instead.